src/maven_tests/testsecprocessor/testsecprocessor.cpp

```diff
@@ -1,13 +1,6 @@
-#ifndef _OFF_T
-#define _OFF_T
-#include <sys/_types.h> /* __darwin_off_t */
-typedef __darwin_off_t          off_t;
-#endif  /* _OFF_T */
-//#define _LARGEFILE_SOURCE
-typedef off_t f_off;
-
 #include "secprocessor.h"
```

Comment:
All of this was no longer necessary after I cleaned up some configuration settings in `testsecprocessor.pro`.